README.md

```diff
@@ -82,6 +82,7 @@
 - [ExtropyIO/defi-bot](https://github.com/ExtropyIO/defi-bot) - Tutorial for building DeFi arbitrage bots.
 - [karmacoma-eth/sending-ether-cheat-sheet](https://gist.github.com/karmacoma-eth/4f206a46dedc6da6808c1ccdef3262d0) - Tips and best practices for sending Ether.
 - [LearnXInY](https://learnxinyminutes.com/docs/solidity/) - Learn Solidity in 15 mins (for experienced devs).
+- [LearnWeb3Dao](http://learnweb3.io/) - A free program that will turn you into a web3 developer. Build a Web3 career the quickest way possible.
```

Comment:
Also too much of a marketing description, let’s please explain in a sentence what the resource provides